contrib/automation_tests/orbit_testing.py

```diff
@@ -6,23 +6,27 @@
 
 import logging
 import time
+import pywinauto
```

Comment:
@ronaldfw : Does this work for you with out this import?

Reply:
I ran all tests without issues. But, yes, should be imported here for sure.